reference/constraints/NotJson.rst

```diff
@@ -0,0 +1,100 @@
+NotJson
```

Comment:
If the related code feature is merged, we'll need to replace `NotJson` by `Json` as the name of the constraint, right?

Reply:
yes, you are right, I've changed the name to `Json` to follow the code.